src/components/modal.js

```diff
@@ -0,0 +1,24 @@
+import A11yDialog from "a11y-dialog";
+
+/**
+ *@param {string } modalClass
+ * @param {string} contentId
+ * @param {function} onShow
+ */
+const init = (modalClass, contentId, onShow = () => {}) => {
+
+  const modalClasses = document.querySelectorAll(`[class*=${modalClass}]`);
```

Comment:
This name is deceiving, the result of documentQuerySelecotorAll is a [NodeList](https://developer.mozilla.org/en-US/docs/Web/API/NodeList), so a more appropriate name would be something like `modalElms`

Reply:
done